imsc1/spec/ttml-ww-profiles.html

```diff
@@ -2679,17 +2679,6 @@ <h4>ittp:progressivelyDecodable</h4>
       <section id='itts-forcedDisplay'>
         <h4>itts:forcedDisplay</h4>
 
-        <p><code>itts:forcedDisplay</code> can be used to hide content whose computed value of <code>tts:visibility</code> is
-        <code>"visible"</code> when the processor has been configured to do so via the application parameter
-        <code>displayForcedOnlyMode</code>.</p>

```

Comment:
Seems a bit stark to delete all the introductory text.

Suggest instead:

```suggestion
        <p><code>itts:forcedDisplay</code> can be used to hide content
        when the processor has been configured to do so via the application parameter
        <code>displayForcedOnlyMode</code>.</p>
```

Reply:
Addressed at https://github.com/w3c/imsc/pull/594/commits/143d722823115d2dec425344da9d0504dd697002